scripts/create_user.py

```diff
@@ -1,7 +1,7 @@
 """ This script creates a new user. Fails if the user already exists. """
 
 import hashlib
-import M2Crypto
+import random
```

Comment:
I think `random` can't be used for password generation. `os.urandom` could be used instead with some modification which would limit vocabulary. 

https://docs.python.org/2/library/random.html
> However, being completely deterministic, it is not suitable for all purposes, and is completely unsuitable for cryptographic purposes.

https://docs.python.org/2/library/os.html#os.urandom
> Return a string of n random bytes suitable for cryptographic use.